src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2484,13 +2484,13 @@
     <value>Open parent folder of searched item</value>
   </data>
   <data name="RefreshItemsDescription" xml:space="preserve">
-    <value>Refresh page contents</value>
+    <value>Reload page contents</value>
   </data>
   <data name="RenameDescription" xml:space="preserve">
-    <value>Rename selected item</value>
+    <value>Change the name of the selected item</value>
```

Comment:
We've been using the term 'Rename' since that start of Files.

Reply:
Same as above, that said, we don't need duplicate strings in the resource file. If there are any duplicates, we can remove the extra one and reuse the string.